test/integration/default/apache_conf_spec.rb

```diff
@@ -1,15 +1,17 @@
 # encoding: utf-8
 
-if os.linux?
+# TODO: do not run those tests on docker yet
+unless ENV['DOCKER']
+  if os.linux?
```

Comment:
we may think of using controls with only_if

> On 16 May 2016, at 17:26, Dominik Richter <notifications@github.com> wrote:
> 
> In test/integration/default/apache_conf_spec.rb:
> 
> > @@ -1,15 +1,17 @@
> >  # encoding: utf-8
> >  
> > -if os.linux?
> > +# TODO: do not run those tests on docker yet
> > +unless ENV['DOCKER']
> > +  if os.linux?
> all of these handlers, including the os.linux? check are creating huge diffs.
> i would like to avoid these in the future. please us:
> 
> return if Env['DOCKER']
> return unless os.linux?
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>